### PR TITLE
Chore/137090995/grunt to gulp

### PR DESCRIPTION
**Pull request by oreoluwade**
**What does this PR do?**

Run repetitive tasks

**Description of Task to be completed?**

Create a gulpfile.js to act as the task runner in place of the existing gruntfile.js while taking note of the specifications in the gruntfile.

**How should this be manually tested?**

run 'gulp' to see that all specified tasks are running.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const browserSync = require('browser-sync');
+const jshint = require('gulp-jshint');
+const mocha = require('gulp-mocha');
+const nodemon = require('gulp-nodemon');
+const sass = require('gulp-sass');
+const bower = require('gulp-bower');
+const jade = require('gulp-jade');
+
+//Default task(s).
+gulp.task('default', ['jshint', 'server', 'watch', 'sass']);
```
Comment (nyambati) [inline gulpfile.js]: This task should be placed below all the tasks
```js
gulp.task('test', ['mochaTest']); 
gulp.task('default', ['jshint', 'server', 'watch', 'sass']);
```

Comment (oreoluwade) [reply gulpfile.js]: Effected the change. Thank you. :)

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+gulp.task('jade', () => {
+    return gulp.src('app/views/**/*.jade')
+        .pipe(jade())
+        .pipe(gulp.dest('public/views/'));
+});
+
+//Bower Task
+gulp.task('bower', () => {
+    bower()
+        .pipe(gulp.dest('./public/lib/'))
```
[the remark this answers is not in the document]
Comment (oreoluwade) [reply gulpfile.js]: Change effected

File: .travis.yml [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
-- '6'
-- '6.1'
-- '5.11'
+  - "6"
+  - "6.1"
+  - "5.11"
+
+services:
+  - mongodb
+  before_script:
```
Comment (nyambati) [inline .travis.yml]: There is a yam syntax error here. `before_script` and services should start from the same line
```yml
services:
  - mongodb
before_script:
```

**Review by nyambati — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 3 above)
Comment (oreoluwade) [reply .travis.yml]: I made the changes you suggested here. Without the "before_script" however.

**Review by nyambati — APPROVED** [review]

**Pull request merged**